src/server/frontend_wayland/wl_mir_window.cpp

```diff
@@ -121,7 +135,9 @@ void WlAbstractMirWindow::commit()
     if (params->size == geometry::Size{})
         params->size = geometry::Size{640, 480};
 
-    params->streams = std::move(std::vector<shell::StreamSpecification>{{mir_surface->stream_id, mir_surface->buffer_offset, {}}});
+    params->streams = std::vector<shell::StreamSpecification>();
```

Comment:
We could use uniform initialization syntax. Vis: `params->streams = std::vector<shell::StreamSpecification>{};`